client/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "backbone": "1.3.3",
-    "backbone.marionette": "2.4.6",
+    "backbone.marionette": "3",
```

Comment:
I prefer we won't update to Marionette 3 as it is a major update with many changes and we don't have tested it enough.